templates/bootstrap/README.md.j2

```diff
@@ -3,3 +3,24 @@
 A Pulp plugin to support hosting your own {{ plugin_dash_short }}.
 
 For more information, please see the [documentation](docs/index.rst) or the [Pulp project page](https://pulpproject.org/).
+
+
+How to File an Issue
+--------------------
+
+[New {{ plugin_dash }} issue](https://pulp.plan.io/projects/{{ plugin_dash }}/issues/new).
```

Comment:
Unfortunately this won't work for all plugins (eg https://pulp.plan.io/projects/ansible_plugin/issues/new)

Reply:
```suggestion
Visit [our tracker](link-to-tracker) to file a {{ plugin_dash }} issue.
```